src/geff/networkx/io.py

```diff
@@ -74,8 +73,6 @@ def write_nx(
             Defaults to 2.
     """
 
-    group = setup_zarr_group(store, zarr_format)
```

Comment:
Does removing this mean we lose the ability do default to zarr 2?

Reply:
I think I fixed this with a commit that I just pushed. It passes the zarr format into write_dicts and then its set correctly there.